### ISSUE TITLE
Products not showing in auto complete

### ISSUE THREAD

**Issue by LouiseMcMahon** [Smile-SA/elasticsuite#219]
Magneto 2.1.2
Elasticsuite latest

I have created a 4 products that are displaying in 2 categories fine they are called prod 1, prod 3, prod 2... their visibility is set to Catalog, Search and product autocomplete is set to 6.

When i search for prod the auto complete only displays my search terms. Do i need to do anything else to get them to display in the auto complete?

**Comment by romainruaud** [comment]
Hello @LouiseMcMahon and thank you for submitting this issue and for using our module.

Did you process a full reindex after creating your 4 products ?

Regards,

**Comment by LouiseMcMahon** [comment]
yes i did

**Comment by romainruaud** [comment]
Are your products assigned to the website ? Do they appear on the catalog search result page ?

**Comment by LouiseMcMahon** [comment]
yes and yes.

**Comment by LouiseMcMahon** [comment]
I enabled the debug log and this is the output

```[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"HEAD","uri":"http://elasticsearch:9200/magento2_default_catalog_product","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.004996} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_product/_stats","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.001482} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"POST","uri":"http://elasticsearch:9200/magento2_default_catalog_product/product//_termvectors?term_statistics=true","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000962} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pr&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000485} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pr&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000258} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000288} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00024} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000209} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00023} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000206} []
[2016-11-28 12:25:26] main.WARNING: Request Failure: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_product/product/_search","headers":{"host":["elasticsearch:9200"]},"HTTP code":400,"duration":0.001331,"error":"query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter"} []
[2016-11-28 12:25:26] main.WARNING: Response ["{\"error\":{\"root_cause\":[{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_product_20161128_120915\",\"line\":1,\"col\":337}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query_fetch\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"magento2_default_catalog_product_20161128_120915\",\"node\":\"0OgXwJToR2eB9W9NzUsoMQ\",\"reason\":{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_product_20161128_120915\",\"line\":1,\"col\":337}}]},\"status\":400}"] []
[2016-11-28 12:25:26] main.ERROR: query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter [] []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"HEAD","uri":"http://elasticsearch:9200/magento2_default_catalog_category","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000382} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_category/_stats","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.001005} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"POST","uri":"http://elasticsearch:9200/magento2_default_catalog_category/category//_termvectors?term_statistics=true","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000802} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pr&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000399} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pr&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000299} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000286} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000263} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000265} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000263} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000245} []
[2016-11-28 12:25:26] main.WARNING: Request Failure: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_category/category/_search","headers":{"host":["elasticsearch:9200"]},"HTTP code":400,"duration":0.001151,"error":"query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter"} []
[2016-11-28 12:25:26] main.WARNING: Response ["{\"error\":{\"root_cause\":[{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_category_20161128_120915\",\"line\":1,\"col\":289}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query_fetch\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"magento2_default_catalog_category_20161128_120915\",\"node\":\"0OgXwJToR2eB9W9NzUsoMQ\",\"reason\":{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_category_20161128_120915\",\"line\":1,\"col\":289}}]},\"status\":400}"] []
[2016-11-28 12:25:26] main.ERROR: query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter [] []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"HEAD","uri":"http://elasticsearch:9200/magento2_default_catalog_product","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000695} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_product/_stats","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.001022} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"POST","uri":"http://elasticsearch:9200/magento2_default_catalog_product/product//_termvectors?term_statistics=true","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000858} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pro&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000354} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pro&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000272} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000269} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000272} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000243} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000266} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000286} []
[2016-11-28 12:25:26] main.WARNING: Request Failure: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_product/product/_search","headers":{"host":["elasticsearch:9200"]},"HTTP code":400,"duration":0.001581,"error":"query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter"} []
[2016-11-28 12:25:26] main.WARNING: Response ["{\"error\":{\"root_cause\":[{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_product_20161128_120915\",\"line\":1,\"col\":337}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query_fetch\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"magento2_default_catalog_product_20161128_120915\",\"node\":\"0OgXwJToR2eB9W9NzUsoMQ\",\"reason\":{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_product_20161128_120915\",\"line\":1,\"col\":337}}]},\"status\":400}"] []
[2016-11-28 12:25:26] main.ERROR: query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter [] []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"HEAD","uri":"http://elasticsearch:9200/magento2_default_catalog_category","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000337} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_category/_stats","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000675} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"POST","uri":"http://elasticsearch:9200/magento2_default_catalog_category/category//_termvectors?term_statistics=true","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00085} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pro&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000305} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=pro&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000283} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00033} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000299} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00032} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00032} []
[2016-11-28 12:25:26] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000313} []
[2016-11-28 12:25:26] main.WARNING: Request Failure: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_category/category/_search","headers":{"host":["elasticsearch:9200"]},"HTTP code":400,"duration":0.001526,"error":"query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter"} []
[2016-11-28 12:25:26] main.WARNING: Response ["{\"error\":{\"root_cause\":[{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_category_20161128_120915\",\"line\":1,\"col\":289}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query_fetch\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"magento2_default_catalog_category_20161128_120915\",\"node\":\"0OgXwJToR2eB9W9NzUsoMQ\",\"reason\":{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_category_20161128_120915\",\"line\":1,\"col\":289}}]},\"status\":400}"] []
[2016-11-28 12:25:26] main.ERROR: query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter [] []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"HEAD","uri":"http://elasticsearch:9200/magento2_default_catalog_product","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.001085} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_product/_stats","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.001472} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"POST","uri":"http://elasticsearch:9200/magento2_default_catalog_product/product//_termvectors?term_statistics=true","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000942} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000508} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000292} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000369} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000397} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000376} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000375} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000382} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000333} []
[2016-11-28 12:25:27] main.WARNING: Request Failure: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_product/product/_search","headers":{"host":["elasticsearch:9200"]},"HTTP code":400,"duration":0.001562,"error":"query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter"} []
[2016-11-28 12:25:27] main.WARNING: Response ["{\"error\":{\"root_cause\":[{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_product_20161128_120915\",\"line\":1,\"col\":320}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query_fetch\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"magento2_default_catalog_product_20161128_120915\",\"node\":\"0OgXwJToR2eB9W9NzUsoMQ\",\"reason\":{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_product_20161128_120915\",\"line\":1,\"col\":320}}]},\"status\":400}"] []
[2016-11-28 12:25:27] main.ERROR: query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter [] []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"HEAD","uri":"http://elasticsearch:9200/magento2_default_catalog_category","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000507} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_category/_stats","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000804} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"POST","uri":"http://elasticsearch:9200/magento2_default_catalog_category/category//_termvectors?term_statistics=true","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000909} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000495} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000385} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000335} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.00035} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prod&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000339} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=product&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000335} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=synonym","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000338} []
[2016-11-28 12:25:27] main.INFO: Request Success: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_thesaurus/_analyze?text=prods&analyzer=expansion","headers":{"host":["elasticsearch:9200"]},"HTTP code":200,"duration":0.000349} []
[2016-11-28 12:25:27] main.WARNING: Request Failure: {"method":"GET","uri":"http://elasticsearch:9200/magento2_default_catalog_category/category/_search","headers":{"host":["elasticsearch:9200"]},"HTTP code":400,"duration":0.001474,"error":"query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter"} []
[2016-11-28 12:25:27] main.WARNING: Response ["{\"error\":{\"root_cause\":[{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_category_20161128_120915\",\"line\":1,\"col\":272}],\"type\":\"search_phase_execution_exception\",\"reason\":\"all shards failed\",\"phase\":\"query_fetch\",\"grouped\":true,\"failed_shards\":[{\"shard\":0,\"index\":\"magento2_default_catalog_category_20161128_120915\",\"node\":\"0OgXwJToR2eB9W9NzUsoMQ\",\"reason\":{\"type\":\"query_parsing_exception\",\"reason\":\"[multi_match] query does not support arrays for the [query] parameter\",\"index\":\"magento2_default_catalog_category_20161128_120915\",\"line\":1,\"col\":272}}]},\"status\":400}"] []
[2016-11-28 12:25:27] main.ERROR: query_parsing_exception: [multi_match] query does not support arrays for the [query] parameter [] []```

**Comment by romainruaud** [comment]
Thank you for the backtrace.

Do you have synonyms and/or expansions registered in the back-office ? 

What is you ElasticSearch version ?

**Comment by LouiseMcMahon** [comment]
Es version is 2.4.2

i have the following synonym `prod,product` which is set to all stores

**Comment by LouiseMcMahon** [comment]
I just disabled the synonym and re indexed and no chnage

**Comment by romainruaud** [comment]
Please rebuild also the "thesaurus" index by running : 

``` bin/magento index:reindex elasticsuite_thesaurus```

**Comment by LouiseMcMahon** [comment]
I ran that and no chnage but when i've been needing a re index i have just been running `bin/magento index:reindex` which rebuilds all the indexes as far as i'm aware.

**Comment by romainruaud** [comment]
Okay, thank you for all those feedbacks. I think we'll be able to fix it. I assign to @afoucret for resolution.

Best regards,

**Comment by afoucret** [comment]
@LouiseMcMahon : you are saying that you are using the last ElasticSuite version. Does this mean the last stable release (2.2.2) or an unstable (master or 2.3.x) ?

**Comment by LouiseMcMahon** [comment]
2.2.2

**Comment by afoucret** [comment]
The fix has been pushed on the 2.2.x branch.
@romainruaud : can you create a release 2.2.3 for this one and previous bugfixes. It has been a while since we don't release any maintenance release.

**Comment by afoucret** [comment]
@romainruaud : I created the release. Please, fix the changelog (`git log 2.2.2..2.2.3`).
@LouiseMcMahon  : you can use 2.2.3 now. Thank for your feedback about this one.

**Comment by romainruaud** [comment]
Since the issue has been fixed in the latest release : https://github.com/Smile-SA/elasticsuite/releases/tag/2.2.3

I close this issue.

Feel free to reopen it if needed, and thank you for reporting it @LouiseMcMahon .

Best regards,
